extensions/schema-compare/src/schemaCompareMainWindow.ts

```diff
@@ -736,7 +736,7 @@ export class SchemaCompareMainWindow {
 			title: loc.compare
 		}).component();
 
-		this.compareButton.onDidClick(async (click) => {
+		this.compareButton.onDidClick(async () => {
```

Comment:
You should be able to also just prefix these with a `_` to have them be ignored for this rule. This is usually preferable since it makes it easier to know what parameters it passes in the future in case you want to update these functions to use them.

Reply:
Oh, neat!